sql/messaging_system_derived/cfr_users_daily_v1/query.sql

```diff
@@ -0,0 +1,48 @@
+WITH windowed AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    impression_id,
+    client_id,
+    ROW_NUMBER() OVER w1_unframed AS _n,
+    --
```

Comment:
```suggestion
    --
    -- Boolean fields that define more specific usage criteria
    (COUNTIF(bucket_id LIKE '%WHATS_NEW%') OVER w1 = 0) AS seen_whats_new
    --
```

This will record whether the user saw whats new on the given day or not.

Reply:
I changed this to `( ... OVER w1 > 0) ... ` in the patch, does that make sense?

Reply:
Yes. You're right. I had inverted the logic.